customizations.md

```diff
@@ -63,6 +63,20 @@ Any system being used to transport images to ECUs needs to modified only to perm
 
 However, in order for an implementation to be Uptane-compliant, no ECU can cause another ECU to install an image without performing either full or partial verification of metadata. This is done in order to prevent attackers from being able to bypass Uptane, and thus execute arbitrary software attacks. Thus, in an Uptane-compliant implementation, an ECU performs either full or partial verification of metadata and images before installing any image, regardless of how the metadata and images were transmitted to the ECU.
 
+
+## Using Uptane with transport security
+Uptane is designed to retain strong security guarantees even in the face of a network attacker.  This includes situations where there either is no transport security or where the transport security is compromised by an attacker.  In this case, Uptane's security loss is that an attacker may disrupt communication between the vehicle and the OEM (e.g., by jamming the signal or dropping packets).  However, malicious packages cannot be installed, mix-and-match attacks are not possible, etc.  This is similar to how a network attacker without key material can cause a TLS connection to fail to connect or disconnect, but cannot compromise the integrity or confidentiality of the connection.
+
+Uptane's security is orthogonal to security systems at other network layers, such as transport security, data link security, etc.  However, there are several reasons why a party may wish to use a security system at the transport layer in coordination with Uptane:
+
+- If a security system at the transport layer is already deployed for other services or is effectively free to deploy, there is little reason not to use it.  For example, it may be beneficial to have authentication provided by a common system for all services in a vehicle, so this may be considered desirable from a uniformity standpoint.
+
+- Regulation may require or suggest that security at the transport layer is provided.  Hence it may be required for non-technical reasons.
+
+- It does not weaken Uptane's security to use Uptane over a transport layer security system.  If the cost is low, then this can be viewed in some situations as adding defense-in-depth, especially if the security system can improve detection, mitigation, or reporting of network disruptions.
+
+- Security at the transport layer may offer forensic proof of origin and destination (when strong mutual authentication is used).  This may be necessary for compliance to OTA update standards and various current draft regulations.
```

Comment:
@iramcdonald edit:
```suggestion
- Security at the transport layer provides forensic proof of origin and destination (when strong mutual authentication is used).  This may be necessary for compliance to OTA update standards and various current draft regulations.
```